sources/platform/console/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Apify Console
+description: Learn about Apify Console's easy account creation and user-friendly homepage for efficient web scraping management.
+sidebar_position: 1
+category: platform
+slug: /console
+---
+
+**Learn about Apify Console's easy account creation and user-friendly homepage for efficient web scraping management.**
+
+---
+
+## Sign Up
+
+To use Apify Console, you need to create Apify account. [Sign up here](https://console.apify.com/sign-up).
+
+![apify-console-sign-up](./images/console-signup.png)
+
+## Log In
+
+To log in to Apify Console, you can use one of the following methods:
+
+* Google Single Sign-On
+* GitHub Single Sign-On
+* The email and password set during account creation
+
+![apify-console-login](./images/console-login.png)
+
+## Homepage Overview
+
+![apify-console-homepage](./images/console-homepage.png)
+
+The Apify Console homepage provides an overview of our services. The header displays your account name and current plan level. The homepage features several sections:
+
+* **Get Started with Apify**: This section guides you to Apify Store. Here, you can explore existing Actors or create your own. A 'Show me how' button directs you to a guided tour on the Console.
+
+* **Recently Viewed**: This section displays Actors you have recently accessed. New users or those who haven't viewed any Actors will see suggestions instead.
+
+* **Suggested Actors for You**: Based on your and other users' recent activities, this section recommends Actors that might interest you.
+
+* **Actor Runs**: This section is divided into two tabs:
+  * **Recent**: View your latest Actor runs.
+  * **Scheduled**: Check your upcoming scheduled runs and tasks.
+
+Use the side menu to navigate other parts of the Apify Console easily.
```

Comment:
Suggestion: This might be a good place to add a hint about keyboard shortcuts.

Reply:
added them in a kind of hacky way, not sure if this is good approach but seems like one that wastes least amount of space

Reply:
Maybe I'd just list one (eg. GH) and then just refer to the shortcut to open the modal of shortcuts?